src/System.Private.CoreLib/src/System/Collections/ObjectModel/ReadOnlyDictionary.cs

```diff
@@ -23,9 +23,9 @@ namespace System.Collections.ObjectModel
     internal class ReadOnlyDictionary<TKey, TValue> : IDictionary<TKey, TValue>, IDictionary, IReadOnlyDictionary<TKey, TValue>
```

Comment:
It doesn't really matter because this is internal, but to match dictionary this should probably have a `where TKey : object` constraint.